partcad/src/partcad/part_factory_obj.py

```
@@ -0,0 +1,152 @@
+        with PartFactoryObj.lock:
+            if (
+                PartFactoryObj.count_inflight_simple < PartFactoryObj.MIN_SIMPLE_INFLIGHT
+                or file_size < PartFactoryObj.MIN_SUBPROCESS_FILE_SIZE
+            ):
+                PartFactoryObj.count_inflight_simple += 1
+                return False
+            else:
+                PartFactoryObj.count_inflight_subprocess += 1
+                return True
+
+    def _load_obj_directly(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_load_obj_directly`

(ANN202)